internal/routes/v1/health.go

```diff
@@ -0,0 +1,11 @@
+package routes
+
+import (
+	"github.com/Pigmice2733/peregrine-backend/internal/handlers"
+	"github.com/gorilla/mux"
+)
+
+// HealthRoutes registers all the routes for health handlers.
+func HealthRoutes(r *mux.Router) {
```

Comment:
Maybe it's just me, but I am always going to go to `/status` before remembering that it is `/health`. What do you think about renaming this?

Reply:
I'd agree with this, it doesn't matter a whole ton but using `/status` would let us send other info along (version, uptime, whatever) that it wouldn't make sense to send on `/health`.

Reply:
I think we should have two seperate endpoints in that case then, `/status` for info, and `/health` as just a health check route.

Reply:
We can add `/status` in the future. `/health` is more of an example handler than anything.